template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/IdentityServiceConstants.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Company.WebApplication1.Identity
+{
+    public class IdentityServiceConstants
```

Comment:
Actually, I changed my mind :smile:. These values will never change for the given app. Also these need to be const to be used in the route attribute (although that may change).